src/main/java/uk/gov/companieshouse/filinghistory/consumer/transformrules/functions/CapitalCaptor.java

```diff
@@ -45,7 +45,7 @@ CapitalCaptures captureCapital(Pattern extract, String altDescription, String so
         while (matcher.find()) {
             Map<String, Integer> namedGroups = matcher.namedGroups();
```

Comment:
This does not need to be passed as a param to the private method and instead can be take straight from the matcher where needed.